index.html

```diff
@@ -7,10 +7,63 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <div class="container">
+        <header>
+            <div class="logo">
+                <img src="../images/logo-cake.png" alt="" srcset="" class="logo-img">
+            </div>
+            <div class="header-side-contents">
+                <div class="hamburger">
+                    <div class="hamburger-line"></div>
+                    <div class="hamburger-line"></div>
+                    <div class="hamburger-line"></div>
+                </div>
+                <h1>The best cakes in the town delivered to your door</h1>
+            </div>
+            
+        </header>
+        
+        <nav>
+            <ul>
+                <li><a href="">Home</a></li>
+                <li><a href="">Cakes</a></li>
+                <li><a href="">Ordering</a></li>
+                <li><a href="">Lessons</a></li>
+                <li><a href="">About</a></li>
+            </ul>
+        </nav>
+        <div class="intro-texts">
+            <h2 class="welcome">Welcome</h2>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Qui sed voluptas, debitis eaque hic nulla laudantium magnam eveniet quam mollitia.</p>
+        </div>
+        
+        <div class="big-image">
+            <!-- img inside this div  -->
+        </div>
+        <div class="small-images">
+            <img src="../images/cake1.jpg" alt="" class="small-img">
+            <img src="../images/cake3.jpg" alt="" class="small-img">
+            <img src="../images/cake4.jpg" alt="" class="small-img">
+            <img src="../images/cake5.jpg" alt="" class="small-img">
```

Comment:
What are the alt tags for these images? make sure you add them, otherwise it won't pass accessibility requirements